src/react/components/pages/predict/predictResult.tsx

```diff
@@ -26,8 +26,15 @@ export default class PredictResult extends React.Component<IPredictResultProps,
         const tagsDisplayOrder = tags.map((tag) => tag.name);
         for (const name of Object.keys(predictions)) {
             const prediction = predictions[name];
```

Comment:
To make code simpler:
const prediction = predictions[name] || {};
...
predictions[name] = prediction;